src/wui/interactive_base.cc

```diff
@@ -1479,6 +1486,12 @@ bool InteractiveBase::handle_key(bool const down, SDL_Keysym const code) {
 void InteractiveBase::cmd_lua(const std::vector<std::string>& args) {
 	const std::string cmd = boost::algorithm::join(args, " ");
 
+	if (chat_provider_) {
+		/** TRANSLATORS: This is a chat message which is automatically sent */
```

Comment:
/** TRANSLATORS: This is a chat message which is automatically sent TRANSLATORS: to all players when a player uses the debug console */